Zend/zend_execute.c

```diff
@@ -583,6 +583,19 @@ static zend_never_inline ZEND_COLD bool zend_wrong_assign_to_variable_reference(
 	return 1;
 }
 
+ZEND_API void zend_cannot_pass_by_reference(uint32_t arg_num)
+{
+	const zend_execute_data *execute_data = EG(current_execute_data);
+	zend_string *func_name = get_function_or_method_name(EX(call)->func);
+	const char *param_name = get_function_arg_name(EX(call)->func, arg_num);
```

Comment:
It looks like these could use the `active` variant as well?

Reply:
I remember that there was a problem with that initially, so I needed to get the function which is going to be called (?). Or `EX(call)` is something different?

Reply:
Oh right, makes sense.